### PR TITLE
Add SciPy elementwise functions poster

### PR DESCRIPTION
**Pull request by mdhaber**
Add "Vectorized Quadrature, Series Summation, Differentiation, Optimization, and Rootfinding in SciPy" poster.

@steppi @tupui please check affiliations and ORCIDs.

### CODE REVIEW

File: presentations/posters/scipy_elementwise/info.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            "orcid": "0000-0003-4806-3601"
+        },
+        {
+            "name": "Albert Steppi",
+            "affiliation": "Quansight Labs",
+            "orcid": "0000-0001-5871-6245"
+        },
+        {
+            "name": "Pamphile Roy",
+            "affiliation": "Consulting Manao GMBH",
```
Comment (tupui) [inline presentations/posters/scipy_elementwise/info.json]:
```suggestion
            "affiliation": "Consulting Manao GmbH",
```

**Review by tupui — COMMENTED** [review]
Thanks Matt!

File: presentations/posters/scipy_elementwise/info.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            "affiliation": "Cal Poly, San Luis Obispo",
+            "orcid": "0000-0003-4806-3601"
+        },
+        {
+            "name": "Albert Steppi",
+            "affiliation": "Quansight Labs",
+            "orcid": "0000-0001-5871-6245"
+        },
+        {
+            "name": "Pamphile Roy",
```
Comment (tupui) [inline presentations/posters/scipy_elementwise/info.json]:
```suggestion
            "name": "Pamphile T. Roy",
```

**Comment by steppi** [comment]
Looks good. Thanks Matt!

**Review by mepa — APPROVED** [review]

**Pull request merged**